.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /.vagrant/
 /ubuntu-xenial-16.04-cloudimg-console.log
+/.idea/
```

Comment:
Why this folder is ignored on Git version control?

What tool will generate this folder?

Reply:
This folder comes from IntelliJ IDEs. I've removed it in 12c2e7d.

Reply:
(Thank you for reviewing this!)